keycloak-themes/moh-app-realm/login/login.ftl

```diff
@@ -60,9 +60,13 @@
         <#if realm.password && social.providers??>
             <div id="kc-social-providers" class="${properties.kcFormSocialAccountContentClass!} ${properties.kcFormSocialAccountClass!}">
                 <ul class="${properties.kcFormSocialAccountListClass!}">
+                <#-- by default all identity providers are shown, different message is displayed in case of IDP outage -->
                     <#list social.providers as p>	
-						<#-- by default all identity providers are shown -->
-						<li class="${properties.kcFormSocialAccountListLinkClass!}"><a style="display: block" href="${p.loginUrl}" id="zocial-${p.alias}" class="zocial ${p.providerId}"> <span>Login with ${p.displayName}</span></a></li>
+                        <#if p.displayName?contains("outage")>
+                            <li class="${properties.kcFormSocialAccountListLinkClass!} idp-outage"><a style="display: block;" href="${p.loginUrl}" id="zocial-${p.alias}" class="zocial ${p.providerId}"> <span>Login with ${p.displayName?remove_ending("outage")} is currently unavailable. IDP team is working on restoring the service.</span></a></li>
```

Comment:
I think it should say "[The] IDP team is working on restoring the service." or preferably just "[The team] is working on restoring the service." 

Users don't care who is working on the problem, and they will not know what IDP stands for.